lib/public/views/LhcFills/Detail/lhcFillDetailsComponent.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+import spinner from '../../../components/common/spinner.js';
+import table from '../../../components/Table/index.js';
+import errorAlert from '../../../components/common/errorAlert.js';
+import { runsActiveColumns } from '../../Runs/ActiveColumns/runsActiveColumns.js';
+import { lhcFillDisplayConfiguration } from './lhcFillDisplayConfiguration.js';
+import { detailsGrid } from '../../../components/Detail/detailsGrid.js';
+
+/**
+ * Component displaying information about a run
```

Comment:
Please update the documentation accordingly

Reply:
Done